frame/nfts/src/lib.rs

```diff
@@ -61,6 +61,26 @@ pub use weights::WeightInfo;
 
 type AccountIdLookupOf<T> = <<T as frame_system::Config>::Lookup as StaticLookup>::Source;
 
+pub trait Incrementable {
+	fn increment(&self) -> Self;
+}
+
+macro_rules! impl_incrementable {
+	($type:ty) => {
+		impl Incrementable for $type {
+			fn increment(&self) -> Self {
+				self.saturating_add(1)
+			}
```

Comment:
@KiChjang Did you think of something like this? Also, should I implement this for all integers, currently I implemented this only for unsigned ones?

Reply:
Yeah, all signed integers could simply just have this trait implemented for them as well.